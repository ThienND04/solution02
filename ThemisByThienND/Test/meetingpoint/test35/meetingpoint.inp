8 14 4
3 6 1
3 4 20
8 4 1
2 3 2
7 1 20
4 5 14
7 3 15
7 1 20
4 3 18
3 5 3
1 6 3
5 2 2
8 6 20
7 5 11
1 2
6 5
9 8
5 8
