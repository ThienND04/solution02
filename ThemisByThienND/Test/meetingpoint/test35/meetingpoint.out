21
75
120
105
